martin-core/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "martin-core"
+version = "0.0.1"
```

let's start with the usual 0.1.0

```suggestion
version = "0.1.0"
```